packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -150,6 +151,12 @@ const createFormDataObject = (data: Record<string, unknown>) => {
 	return formData;
 };
 
+const bodyToString = async (body: Buffer | Readable) =>
```

Comment:
since this can be used for more than just request/response bodies (i.e. binary data, data from the filesystem, etc), we should rename this.